rfi_file_monitor/applicationwindow.py

```diff
@@ -490,6 +519,26 @@ def operations_button_cb(self, button):
         self._operations_box.pack_start(new_operation, False, False, 0)
         new_operation.show_all()
         self.update_monitor_switch_sensitivity()
+        self.controls_operations_live.append([_class.NAME, _class])
+        self.controls_operations_live_combo.set_model(self.controls_operations_live)
+
+    def remove_operations_button_cb(self, button):
+        _class = self.controls_operations_live_combo.get_model()[self.controls_operations_live_combo.get_active_iter()][1]
+        self.controls_operations_live.remove(self.controls_operations_live_combo.get_active_iter())
+        self.controls_operations_live_combo.set_model(self.controls_operations_live)
+        ops = self._operations_box.get_children()
+        for op in ops:
+            if op.NAME == _class.NAME:
+                op_to_remove = op
+                break
+        old_index = op_to_remove.index
+        self._operations_box.remove(op_to_remove)
+        self._operations_box.resize_children()
+
+
+        for op in self._operations_box.get_children():
```

Comment:
The `get_children()` here is redundant: iterating a Gtk widget goes automatically across its children.